content/en/blog/falcosidekick-knative.md

```diff
@@ -0,0 +1,463 @@
+---
+title: Falcosidekick + Knative = a Kubernetes Response Engine, Part 3
+date: 2021-04-28
+author: Scott Nichols and Dan Papandrea
+slug: falcosidekick-knative
+---
+
+As the Cloud Native ecosystem grows and the idea that an integrator can browse
+the offerings and slap them together like an a la carte menu resonates. We call
+this _Thinking Cloud Native_.
```

Comment:
slug: falcosidekick-reponse-engine-part-1-kubeless
---

> *This blog post is part of a series of articles about how to create a `Kubernetes` response engine with `Falco`, `Falcosidekick` and a `FaaS`.*
> 
> See other posts:
> * [Kubernetes Response Engine, Part 1 : Falcosidekick + Kubeless]({{< ref "/blog/falcosidekick-reponse-engine-part-1-kubeless" >}})
> * [Kubernetes Response Engine, Part 2 : Falcosidekick + OpenFaas]({{< ref "/blog/falcosidekick-reponse-engine-part-2-openfass" >}})

----

Reply:
Knative is not a `FaaS`.

Reply:
I'm working on creating my PR for part 4 right now and I'm not using a faas ether.
What should we call it? 

"This blog post is part of a series of articles about how to create a Kubernetes response engine with Falco, Falcosidekick and a"
* third part tool
* external api

None of those are very a peeling words...
At least lets call it something generic since we will soon have a 5th one that uses argocd workflow.

Reply:
@n3wscott: sorry, wrong copy/paste, but you get the idea 😉

Reply:
The current example of "workload platform" sounds good to me.

Reply:
> This blog post is part of a series of articles about how to create a Kubernetes response engine with Falco, Falcosidekick and a workload platform (like Serverless or FaaS)

?